controllers/certificaterequest/certificaterequest_controller.go

```diff
@@ -390,14 +390,18 @@ func (r *CertificateRequestReconciler) createCertificateSecret(reqLogger logr.Lo
 
 // revokeCertificateAndDeleteSecret revokes certificate if it exists
 func (r *CertificateRequestReconciler) revokeCertificateAndDeleteSecret(reqLogger logr.Logger, cr *certmanv1alpha1.CertificateRequest) error {
-	//todo - actually delete secret when revoking
-
-	if SecretExists(r.Client, cr.Spec.CertificateSecret.Name, cr.Namespace) {
+	exists, err := SecretExists(r.Client, cr.Spec.CertificateSecret.Name, cr.Namespace)
```

Comment:
Do we want to keep the todo here since we're still not deleting any secret after revoking?